src/datadog/grok/parse_grok.rs

```diff
@@ -416,6 +420,24 @@ mod tests {
+            ],
+            BTreeMap::new()
+        ).expect("couldn't parse rules");
+
+        let parsed = parse_grok(
```

Comment:
Nit: this scary text could live under a `parse_grok` directory in https://github.com/vectordotdev/vrl/tree/main/tests/data/fixtures. Same for the pattern above.

Reply:
good idea ! https://github.com/vectordotdev/vrl/pull/1079/commits/b3727585c449a0da95d66e9b65f2c3ec3f3bf998